drake/examples/Quadrotor/Quadrotor.m

```diff
@@ -113,13 +113,13 @@ function visualizePlan(x,lcmgl)
   methods (Static)
     
     function runOpenLoop
-      r = Quadrotor('lidar');
+      r = Quadrotor();
```

Comment:
> i really want to leave the lidar on so it’s getting coverage

I agree.  

I think I made this change early on while hacking around in the quadrotor examples.  In any case, this PR doesn't even call Quadrotor.runOpenLoop so it shouldn't be here.